packages/bpk-component-chip-group/src/BpkChipGroup.module.scss

```diff
@@ -42,10 +42,6 @@
   &--wrap {
     flex-wrap: wrap;
   }
-
-  &--with-label {
-    padding-inline-start: tokens.bpk-spacing-base();
```

Comment:
We decided page (or some outer consumer container) will always have padding so we just align to that